packages/grafana-data/src/datetime/timezones.ts

```diff
@@ -0,0 +1,390 @@
+// List taken from https://stackoverflow.com/questions/38399465/how-to-get-list-of-all-timezones-in-javascript
+
+export const getTimeZoneGroups = () => {
+  return [
+    { label: 'Africa', options: africaZones },
+    { label: 'America', options: americaZones },
+    { label: 'Antarctica', options: antarcticaZones },
+    { label: 'Asia', options: asiaZones },
+    { label: 'Atlantic', options: atlanticZones },
+    { label: 'Australia', options: australiaZones },
+    { label: 'Europe', options: europeZones },
+    { label: 'Indian', options: indianZones },
+    { label: 'Pacific', options: pacificZones },
+  ];
+};
+
+const europeZones = [
```

Comment:
Can you move these into the getTimeZoneGroups function so we do not need to create these arrays in global scope?